driver-core/src/main/com/mongodb/client/model/changestream/OperationType.java

```diff
@@ -48,7 +46,21 @@
     /**
      * The invalidate operation type
      */
-    INVALIDATE("invalidate");
+    INVALIDATE("invalidate"),
+
+    /**
+     * The drop operation type
+     *
+     * @since 3.8.2
+     */
+    DROP("drop"),
```

Comment:
Controversial to add new operation types in a point release but this is essentially a bug fix release.